content/en/docs/rules/supported-fields.md

```diff
@@ -41,10 +42,11 @@ Name | Type | Description
 `evt.pluginname` | CHARBUF | if the event comes from a plugin, the name of the plugin that generated it. The plugin must be currently loaded.
 `evt.plugininfo` | CHARBUF | if the event comes from a plugin, a summary of the event as formatted by the plugin. The plugin must be currently loaded.
 
-### Field Class: evt (for system calls)
+### Field Class: evt
```

Comment:
I like the outline to have the `(for system calls)` part so that it's unambiguous on the right but I don't want this to be a manual thing to do. So I can support that by making the one liner a bit more ugly. Do you agree?

Reply:
Yeah I agree, I see your point!

Reply:
updated accordingly 👍